app/models/grda_warehouse/hud/client.rb

```diff
@@ -1403,9 +1406,15 @@ def presented_with_family?(after: nil, before: nil)
       end
     end
 
+    def pii(user: nil)
```

Comment:
I wonder if this should be named `masked_pii` or `handle_pii`.  I think `pii` is fine, but doesn't imply that it does any manipulation.

Reply:
I like pii_accessor and pii_provider